### PR TITLE
[f-2] chore: update references, and refactor data handling

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR:

- updates App references to BreakoutApp and SidebarApp directly
- refactors data handling to pass the correct url back to SFCC data API

This creates a working e2e flow for selecting the image and having the data pass back correctly and stored in the custom attribute, and then rendered on the page

### Stacked PR Chain: f-2
| PR | Title | Status |  Merges Into  |
|:--:|:------|:-------|:-------------:|
|#90|👉 chore: update references, and refactor data handling|Pending|-|
|#92|chore: remove unnecessary files|Pending|#90|

### CODE REVIEW

File: frontend/src/components/AssetBrowser/AssetBrowserContainer.tsx [hunk 1]
```diff
@@ -204,8 +206,24 @@ export class AssetBrowserContainer extends Component<Props, State> {
...
+
+    const src = `https://${sourceDomain}${assetData.attributes.origin_path}`;
+
+    const data = {
+      src,
+    };
+
+    this.props.onSelectAsset(data);
+  };
+
```
Comment (luqven) [inline frontend/src/components/AssetBrowser/AssetBrowserContainer.tsx:225]: A little confused by this. Why not handle the domain parsing inside AssetBrowser, where [`paseSourceDomain()` ](https://github.com/imgix/sf-commerce-cloud/blob/4294bddce19c83d897b12d9f832ccf302a930dff/frontend/src/components/AssetBrowser/AssetBrowser.tsx#L103)already exists? Is this a limitation of needing to define the callback at the container level?

File: frontend/src/components/grids/AssetGrid.tsx [hunk 2]
```diff
@@ -19,19 +23,23 @@ export function AssetGrid({
...
-                handleAssetGridClick({
-                    src: asset
-                })
+              handleAssetGridClick({
+                src: asset,
+              });
             }
-        }}>
+          }}
+        >
```
Comment (luqven) [inline frontend/src/components/grids/AssetGrid.tsx:42]: I must not have the same style-on-save ruleset as you. Your change is def better, but struggling to understand why I'm not formatting the same as you on my end. We should take a minute to figure this out to make unnecessary diffs like this a thing of the past.

File: frontend/src/index-sidebar.tsx [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { App } from "./components";
+import { App as SidebarApp } from "./components/SidebarApp";
```
Comment (luqven) [inline frontend/src/index-sidebar.tsx:3]: A little OOL here. How are now telling the build script which FE app to build?

File: frontend/src/components/AssetBrowser/AssetBrowserContainer.tsx (hunk 1 above)
Comment (frederickfogerty) [reply frontend/src/components/AssetBrowser/AssetBrowserContainer.tsx:225]: I guess I never replied to this 🤦 . To be honest there wasn't a motivation behind putting it here, if you think it lives better in `AssetBrowser` I can refactor it to be there.

File: frontend/src/components/grids/AssetGrid.tsx (hunk 2 above)
Comment (frederickfogerty) [reply frontend/src/components/grids/AssetGrid.tsx:42]: I'm using prettier for formatting - what are you using?

File: frontend/src/index-sidebar.tsx (hunk 3 above)
Comment (frederickfogerty) [reply frontend/src/index-sidebar.tsx:3]: We resolved this offline

**Pull request merged**